selfdrive/car/honda/carcontroller.py

```diff
@@ -101,6 +101,10 @@ def update(self, enabled, CS, frame, actuators,
       # send pcm acc cancel cmd if drive is disabled but pcm is still on, or if the system can't be activated
       pcm_cancel_cmd = True
 
+    # Never send cancel command if we never enter cruise state (no cruise if pedal)
+    # Cancel cmd causes brakes to release at a standstill causing grinding
+    pcm_cancel_cmd &= CS.CP.enableCruise
```

Comment:
```suggestion
    pcm_cancel_cmd = pcm_cancel_cmd and CS.CP.enableCruise
```

Probably best to not use bitwise and to prevent weird issues later.